packages/lodestar-cli/src/validatorDir/paths.ts

```diff
@@ -14,13 +13,13 @@ export const LOCK_FILE = ".lock";
 // Dynamic paths computed from the validator pubkey
 
 export function getValidatorDirPath(
-  {keystoresDir, pubkey, prefixed}: {keystoresDir: string; pubkey: string; prefixed?: boolean}
```

Comment:
Wasn't this causing errors because we don't have 0x prefix when using some command?

Reply:
The rest of the code expects this path prefixed with `0x`. Is the current Keystore returning the pubkey prefixed?

Reply:
The new bls-keystore library returns the pubkey without a prefix, which is why I made this change (bear in mind that this was upgraded from 1.0.0-beta8 to 2.0.0 so a number of things have changed since we last upgraded that pkg here)

Reply:
It seems all the hex strings in the new keystore library are non-prefixed. And that is actually a good thing for cross-client compatability, other clients don't use 0x prefix in keystores, including eth2.0-deposit-cli.

In that case, we should keep this logic in this function so we can add prefixes asap in this part of the codebase.

Reply:
ok, just reverted paths.ts and format.ts